### PR TITLE
api/block: driveby cleanups

### PR DESCRIPTION
**Pull request by davecheney**
Multi line logging, seriously ??

### CODE REVIEW

File: api/block/client.go [hunk 1]
```diff
@@ -69,12 +64,9 @@ func (c *Client) SwitchBlockOff(blockType string) error {
...
-	result := params.ErrorResult{}
+	var result params.ErrorResult
 	if err := c.facade.FacadeCall("SwitchBlockOff", args, &result); err != nil {
 		return errors.Trace(err)
 	}
-	if result.Error != nil {
-		return result.Error
-	}
-	return nil
+	return errors.Trace(result.Error)
```
Comment (mjs) [inline api/block/client.go]: The original code was weird here!

**Comment by mjs** [comment]
LGTM

**Comment by davecheney** [comment]
$$merge$$

**Comment by davecheney** [comment]
$$merge$$

**Comment by davecheney** [comment]
$$__JFDI__$$

**Comment by davecheney** [comment]
$$__JFDI__$$

**Comment by davecheney** [comment]
$$__JFDI__$$

**Pull request merged**